website/docs/guides/migration/versions/04-upgrading-to-v1.4.md

```diff
@@ -8,39 +8,44 @@ description: New features and changes in dbt Core v1.4
 - [CLI Installation guide](/docs/get-started/installation)
 - [Cloud upgrade guide](/docs/dbt-versions/upgrade-core-in-cloud)
 
+**Planned final release:** January 2023
+
+dbt Core v1.4 is a "behind-the-scenes" release. We've been hard at work rebuilding `dbt-core` internals on top of more-solid foundations, to enable an exciting year of new feature development. Check out the [v1.5 milestone](https://github.com/dbt-labs/dbt-core/milestone/82) in GitHub for a preview of what's planned for April.
+
 ## What to know before upgrading
 
 dbt Labs is committed to providing backward compatibility for all versions 1.x. If you encounter an error upon upgrading, please let us know by [opening an issue](https://github.com/dbt-labs/dbt-core/issues/new).
 
 ### For consumers of dbt artifacts (metadata)
 
-The manifest schema version has updated to `v8`.
-These changes are relevant for people who:
-- parse or analyze the contents of the `manifest.json` file
-- have custom code accessing the [`model`](https://docs.getdbt.com/reference/dbt-jinja-functions/model) or [`graph`](https://docs.getdbt.com/reference/dbt-jinja-functions/graph) variables. For example:
+The manifest schema version has updated to `v8`. These changes are relevant for people who parse or analyze the contents of the `manifest.json` file, or who have custom code accessing the [`model`](https://docs.getdbt.com/reference/dbt-jinja-functions/model) or [`graph`](https://docs.getdbt.com/reference/dbt-jinja-functions/graph) variables, e.g. `{{ model.root_path }}`.
 
-```
-{{ model.root_path }}
-```
-
-Relevant changes are: 
-- The `root_path` attribute is removed for all nodes to reduce duplicative information.
+Relevant changes:
+- The `root_path` attribute has been removed for all nodes to reduce duplicative information.
 - Unused attributes have been removed from seed `nodes`, including `depends_on`, and from `macros`, including `tags`.
 - The `unique_id` of docs blocks now start with `doc` for consistency with other resource types.
 
+### For maintainers of adapter plugins
+
+Link to GH discussion forthcoming
```

Comment:
TODO for @dataders (ty!)

Reply:
To be clear - I'm be okay with merging this PR as is, and @dataders can update later

Reply:
it'll be up in the next hour! i say merge away.